shlr/tcc/tccpp.c

```diff
@@ -19,7 +19,9 @@
  */
 
 #include "tcc.h"
-
+#ifdef _MSC_VER
```

Comment:
there's no problem to include this on linux, i prefer to avoid unnecessaries ifdefs